File: CS/Q251885/MainForm.cs
using System;
using DevExpress.XtraEditors;

namespace Q251885 {
    public partial class MainForm :XtraForm {
        public MainForm() {
            InitializeComponent();
        }

        private void MainForm_Load(object sender, EventArgs e) {
            // TODO: This line of code loads data into the 'nwindDataSet.Categories' table. You can move, or remove it, as needed.
            this.categoriesTableAdapter.Fill(this.nwindDataSet.Categories);

        }

        private void OnSwitchViewButtonClick(object sender, EventArgs e) {
            if (categoriesGrid.MainView == categoriesViewSimple)
                categoriesGrid.MainView = categoriesViewLayout;
            else categoriesGrid.MainView = categoriesViewSimple;
        }
    }
}

File: VB/Q251885/MainForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors

Namespace Q251885
	Partial Public Class MainForm
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub MainForm_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'nwindDataSet.Categories' table. You can move, or remove it, as needed.
			Me.categoriesTableAdapter.Fill(Me.nwindDataSet.Categories)

		End Sub

		Private Sub OnSwitchViewButtonClick(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			If categoriesGrid.MainView Is categoriesViewSimple Then
				categoriesGrid.MainView = categoriesViewLayout
			Else
				categoriesGrid.MainView = categoriesViewSimple
			End If
		End Sub
	End Class
End Namespace